### PR TITLE
@thunderstore/cyberstorm: cleanup for Header component

### PR DESCRIPTION
**Pull request by anttimaki**

### CODE REVIEW

**Comment by anttimaki** [comment]
Current dependencies on/for this PR:
* master
  * **PR #831** <a href="https://app.graphite.dev/github/pr/thunderstore-io/thunderstore-ui/831" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/thunderstore-io/thunderstore-ui/831?utm_source=stack-comment).

File: packages/cyberstorm/src/components/Header/Header.tsx [hunk 1]
```diff
@@ -1,50 +1,19 @@
...
+import styles from "./Header.module.css";
+import { DevelopersDropDown } from "./DevelopersDropDown";
+import { PlainButton } from "../Button/Button";
+import { CommunitiesLink, IndexLink, PackageUploadLink } from "../Links/Links";
+import { Tooltip } from "../Tooltip/Tooltip";
 import { ThunderstoreLogo } from "../../svg/svg";
-import { Tooltip } from "../..";
-import { DapperInterface, useDapper } from "@thunderstore/dapper";
-import { usePromise } from "@thunderstore/use-promise";
+import { UserDropDown } from "./UserDrowDown";
```
Comment (Oksamies) [inline packages/cyberstorm/src/components/Header/Header.tsx:11]: File name is drowning 🤽

**Review by MythicManiac — APPROVED** [review]
Looks good to me

**Review by Oksamies — CHANGES_REQUESTED** [review]

File: packages/cyberstorm/src/components/Header/DevelopersDropDown.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
+"use client";
```
[the remark this answers is not in the document]
Comment (anttimaki) [reply packages/cyberstorm/src/components/Header/DevelopersDropDown.tsx]: No need, removed.

File: packages/cyberstorm/src/components/Header/Header.tsx (hunk 1 above)
Comment (anttimaki) [reply packages/cyberstorm/src/components/Header/Header.tsx]: Baywatched.

**Review by Oksamies — APPROVED** [review]

**Pull request merged**